docs/pgp-workaround.md

```diff
@@ -0,0 +1,58 @@
+# PGP fallback URL in air-gapped environment workaround
+
+## Reminders
+
+When Elastic Agent tries to perform an upgrade, it verifies first the binary signature with key bundled in the Agent.
+This process has a backup mechanism that will use the key coming from https://artifacts.elastic.co/GPG-KEY-elastic-agent instead of the one he already has.
```

Comment:
```suggestion
When Elastic Agent tries to perform an upgrade, it first verifies the binary signature with the key bundled in the Agent.
This process has a backup mechanism that will use the key coming from https://artifacts.elastic.co/GPG-KEY-elastic-agent instead of the one you already have.
```